### ISSUE TITLE
Heatplots

### ISSUE THREAD

**Issue by osorensen** [ocbe-uio/BayesMallows#255]
Good suggestion from Marta: add a function to create heat plots as in Figure 5 of the [JMLR paper](https://jmlr.org/papers/v18/15-481.html).

**Comment by osorensen** [comment]
I've now implemented a `heat_plot()` function which does this. It uses the consensus ranking to order the items along the x-axis.

``` r
library(BayesMallows)
model_fit <- compute_mallows(potato_visual, nmc = 10000, seed = 1)
model_fit$burnin <- 1000

heat_plot(model_fit)
```

![](https://i.imgur.com/x91aAtl.png)

``` r

# Items are ordered along the horizontal axis according to the ordering
# returned by compute_consensus, whose default argument is type="CP".

heat_plot(model_fit, type = "MAP")
```

![](https://i.imgur.com/3zipT5U.png)

<sup>Created on 2023-03-10 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by osorensen** [comment]
Closed by #288
